### PR TITLE
fix(transaction): update the redirect after transaction cancellation

### PR DESCRIPTION
**Pull request by andreybakanovsky**
It takes place when a spend transaction is canceled/deleted. The app tries to redirect us to the virtual account which is called 'store'.

Resolved this error:
image

### CODE REVIEW

**Comment by ka8725** [comment]
@andreybakanovsky please explain these points in the PR description:

- Why and when do these errors happen? Are they considered an error?
- What's the suggested solution from the user's point of view?
- Are there alternative solutions? Are there more accessible solutions?
- Is this kind of error specific to accounts only? What to do with the other pages?

File: app/controllers/transactions_controller.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ def show; end
   def destroy
     transaction = Transaction.find(ps.fetch(:id)) # TODO: implement authorization
     transaction.destroy
-    redirect_to account_path(transaction.to_account)
+    if transaction.to_account.name == 'store'
+      redirect_to account_path(transaction.from_account)
+    else
+      redirect_to account_path(transaction.to_account)
+    end
```
Comment (ka8725) [inline app/controllers/transactions_controller.rb:11]: Add tests?

Comment (andreybakanovsky) [reply app/controllers/transactions_controller.rb:11]: Unquestionably

**Review by VladislavSokov — APPROVED** [review]

**Comment by andreybakanovsky** [comment]
I deliberately missed a few questions, we have to move on.

**Pull request merged**